.rubocop.yml

```diff
@@ -1,6 +1,6 @@
 Documentation:
   Enabled: false
 LineLength:
-  Max: 99
+  Max: 170
```

Comment:
I'm not a fan of changign the rubocop.yml. It's there a way to fix the method having issues with the line length? I'm pretty sure that if you add a enter in the middle of the line and then execute: `bundle exec rubocop app spec -R --format simple -a` it will fix the error automatically

Reply:
Haha, don't do that